.github/workflows/checks.yml

```
@@ -83,17 +83,29 @@ jobs:
         name: Install (dev)
         run: |
           pip install -r requirements-dev.txt
+
       - if: matrix.install-mode == 'standard'
         name: Install (standard)
         run: |
           pip install "git+${{ format('{0}/{1}@{2}', github.server_url, github.repository, github.ref) }}"
+      - name: Install (idaes-solvers)
+        run: |
+          idaes get-extensions
       - if: matrix.coverage
         name: Enable coverage for pytest
         run: echo PYTEST_ADDOPTS="$PYTEST_ADDOPTS --cov --cov-report=xml" >> $GITHUB_ENV
       - name: Run pytest
         run: |
           pip install pytest  # ensure pytest is installed (should do nothing if already present from requirements-dev.txt)
           pytest --pyargs reaktoro_pse --verbose
+      - name: Install Jupyter kernel
+        run: |
+          jupyter kernelspec list
+          python -m ipykernel install --user --name "${{ env.KERNEL_NAME }}"
+          jupyter kernelspec list
+      - name: Run pytest with nbmake
+        run: |
+          pytest --nbmake --nbmake-kernel="${{ env.KERNEL_NAME }}"  
```

I was able to confirm that `**/*.ipynb` doesn't work here to run pytest with nbmake, unlike in `watertap-org/watertap` (possibly because of the different directory structure?). Passing the path to the directory containing the notebooks more explicitly seems to work:

```suggestion
          pytest --nbmake --nbmake-kernel="reaktoro-pse-dev" src/reaktoro_pse/tutorials/*.ipynb
```